lib/mail_room/configuration.rb

```diff
@@ -8,17 +8,20 @@ class Configuration
     def initialize(options={})
       self.mailboxes = []
 
-      if options.has_key?(:config_path)
-        config_file = YAML.load_file(options[:config_path])
+      return unless options.has_key?(:config_path)
+      return unless File.exist?(options[:config_path])
+      
+      config_file = YAML.load_file(options[:config_path])
 
-        set_mailboxes(config_file[:mailboxes])
-      end
+      set_mailboxes(config_file[:mailboxes])
     end
 
     # Builds individual mailboxes from YAML configuration
     # 
     # @param mailboxes_config
     def set_mailboxes(mailboxes_config)
+      return unless mailboxes_config
```

Comment:
I'd rather not just silently return, having an error raise here tells the user that they've misconfigured. Granted, a better experience would be to rescue that error and provide a helpful message. But that's a separate issue.